docs/source/index.rst

```diff
@@ -21,6 +21,7 @@ Daml Documentation
    Building Your App <getting-started/index>
    getting-started/app-architecture
    getting-started/first-feature
+   getting-started/testing
```

Comment:
you also need to update the pdf version of this in `docs/configs/pdf/index.rst`.

Reply:
and mirror those changes in the assembly repo.

Reply:
I had a look at what's in the assembly repo. It appears there's a shorter version of the table of contents. What needs to be replicated? Where is this process documented? Who owns it?

Reply:
In https://github.com/DACH-NY/assembly/blob/main/docs/index/index_html.rst we need to add it to the getting started guide, similar for https://github.com/DACH-NY/assembly/blob/main/docs/index/index_pdf.rst.

Do you have an idea of where this should be documented such that people actually read it? Maybe just as a comment in this file? In the readme in the assembly repo? Somewhere else?

Reply:
Could `docs/README.md` be a good place for these pointers to be?

Btw my question was not about where we need to fill the table of contents, but what needs to be there. It looks like a shorter version of the original table of contents but I cannot tell what the rationale is to include or not include something.

Reply:
In the end, the one in this repo is only to get something for easy testing. The authorative ToC is only in the assembly repo the one here is not used at all in the final build. The one in the assembly repo is just structured differently, it still includes everything but sometimes at a deeper level (if something is unreachable from the toc sphinx throws an error).

i’ll try to add something to the readme later today.

Reply:
Thanks. How can I test whether the documentation in the assembly repo produces the artifact I would expect?

Reply:
https://github.com/DACH-NY/assembly/blob/main/docs/README.md but that currently only describes how to deal it with a released version. It is possible to test against a branch but that is currently undocumented and very hacky.